### DISCUSSION TITLE
My feedbacks

### DISCUSSION THREAD

**Discussion by ghost** [phcode-dev/phoenix#1763] [Ideas]
1. Safari support will be appreciated.
2. URL identifier for every project could be useful.
3. when i create a canvas game i saw that document and window is not defined in the jshint errors. can they be fixable.
4. image upload

**Comment by abose** [comment]
Hi @volcanblanco thanks for the feature requests.

1. Safari support will come in this release or the next.
2. Since all files are local, we dont have a concept of project urls, but for future cloud projects, we may have that.
3. The game project is meant for schools which use phcode to engage students with HTML and cam have errors. It is not actually literal games. It is part of a user submitted public project, details of which can be found in that project index.html file.
4. Will be added for cloud projects. For local projects, just copy the age to the local project folder

**Reply by abose** [reply]
If you have any further ideas, please drop it in new duscussion threads to help improve phcode.

**Reply by ghost** [reply]
Great. Thanks a lot.
